helm/charts/kratos/templates/statefulset-mail.yaml

```diff
@@ -115,10 +114,10 @@ spec:
         {{- if $extraContainers }}
 {{ tpl $extraContainers . | indent 8 }}
         {{- end }}
-      {{- if $extraInitContainers }}
+        {{- if .Values.deployment.extraInitContainers }}
       initContainers:
-{{ tpl $extraInitContainers . | indent 8 }}
-      {{- end }}
+{{- tpl .Values.deployment.extraInitContainers . | nindent 8 }}
```

Comment:
This change will remove the possibility of specifying statefulset only init containers, please restore the ternary operation :)